test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Csv.Tests.ps1

```diff
@@ -48,7 +48,7 @@ Describe "ConvertTo-Csv" -Tags "CI" {
 
     It "Should output an array of objects" {
         $result = $testObject | ConvertTo-Csv
-        ,$result | Should -BeOfType "System.Array"
+        , $result | Should -BeOfType "System.Array"
```

Comment:
Why do we need the change? 
Also using "," is very strange here.

Reply:
I think that's just a change from VS Code's auto-formatter.

The unary comma is used to ensure that `Should` sees the whole collection, rather than the unwrapped series of pipeline objects. Probably `Should -InputObject $result -BeOfType [array]` would be a more explicit approach, but it does read a bit funny either way. 🙂

Reply:
It seems it is fake test:
```
(,1).GetType()

IsPublic IsSerial Name                                     BaseType
-------- -------- ----                                     --------
True     True     Object[]                                 System.Array

Reply:
The pipeline removes the wrapper, though.

```PowerShell
, [Int[]](1,2) | ForEach-Object GetType
```

Reply:
The `,` prefix forces that to be an array, so this test will never fail?  In this particular case, maybe it's better to check $result.GetType()?

Reply:
```suggestion
        $result.GetType() | Should -Be [Array]
```